src/raw_to_processed.py

```diff
@@ -0,0 +1,222 @@
+import os
+import pandas as pd
+from glob import glob
+from helpers import load_cfg
+from minio import Minio
+
+###############################################
+# Parameters & Arguments
+###############################################
+DATA_PATH = "data/"
+YEARS = ["2022", "2023"]
+TAXI_LOOKUP_PATH = "src/data/taxi_lookup.csv"
+CFG_FILE =  "config/datalake.yaml"
+###############################################
+
+
+###############################################
+# Utils
+###############################################
+def create_bucket(bucket_name):
```

Comment:
This code snippet has been used over and over again. Consider include it into an `utils` file

Reply:
Fixed!
I have created a file named minio_utils.py in utils/ containing functions (connect, create bucket, etc.) related to MinIO to facilitate usage and debugging.